tools/debian/rules

```diff
@@ -43,3 +59,23 @@ override_dh_install:
 
 	dh_install -Xusr/src/debug
 	make install-tests DESTDIR=debian/cockpit-tests
+
+ifeq ($(C_BRIDGE),)
+	mkdir -p debian/cockpit-bridge/usr/share/cockpit/python/
+	mv debian/tmp/usr/lib/python*/*-packages/cockpit* debian/cockpit-bridge/usr/share/cockpit/python/
```

Comment:
@allisonkarlitskaya: The [Debian policy wrt. package names](https://www.debian.org/doc/packaging-manuals/python-policy/index.html#module-package-names) seemed pretty clear to me. However, I checked [Contents-all.gz](http://ftp.de.debian.org/debian/dists/unstable/main/Contents-all.gz) and there is indeed a lot of packages which use private python modules, e.g. [d-feet](https://packages.debian.org/sid/all/d-feet/filelist). 

This seems to directly contradict [3.6 "Module Path"](https://www.debian.org/doc/packaging-manuals/python-policy/index.html#module-package-names):

> Directories with private Python modules must be absent from the sys.path. 

Most packages with private python modules install them into /usr/share/*packagename* as per spec, see

    zgrep '^usr/share/.*__init__.py' /tmp/Contents-all.gz  | awk '{print $2}' | sort -u

I checked a few packages with private modules, but they just have static executables, no pip-auto-generated ones. So I didn't yet find one which mangles sys.path.

I can't say I like this much, but I don't want to hold this up -- this is also something that we can change later if it leads to bug reports or problems.

Reply:
.. in other words, I changed this PR to install it as a public Python module inside of cockpit-bridge now.